BynderExtension/Workers/AssetDownloadWorker.cs

```diff
@@ -39,31 +40,56 @@ public void Execute(Entity resourceEntity)
             if (asset == null)
             {
                 _inRiverContext.Log(LogLevel.Error, "Asset information is empty");
+
+                bynderDownloadStateField.Data = BynderStates.Error;
+                _inRiverContext.ExtensionManager.DataService.UpdateFieldsForEntity(new List<Field> { bynderDownloadStateField });
```

Comment:
This prevents this to be executed over and over again and erroneous Assets/Resources to be identified.